.github/actions/setup-node/action.yml

```diff
@@ -3,11 +3,12 @@ description: Sets up Node and Build SDK
 
 inputs:
   node-version:
+    description: "Specify Node version"
     required: false
-    default: '16'
+    default: "lts/*"
```

Comment:
Every October this may surprise us :)
Let's use a fixed version (v22 for example).